test/e2e/specs/i18n/i18n__gutenboarding.ts

```diff
@@ -39,13 +39,7 @@ describe( 'I18N: Gutenboarding', function () {
 
 			it( `Display translations on design selector page (${ locale })`, async function () {
 				await validatePageTranslations( page, locale );
-				await gutenboardingFlow.selectDesign( 'Stratford' );
-			} );
-
-			it( `Display translations on style preview page (${ locale })`, async function () {
-				await gutenboardingFlow.selectFont( 'Raleway' );
-				await validatePageTranslations( page, locale );
-				await gutenboardingFlow.clickNextButton();
+				await gutenboardingFlow.selectDesign( '' );
```

Comment:
This selects the 'Blank Canvas' right? If so, could we perhaps put 'Blank Canvas'?

Reply:
Yes, it selects the first design option, which currently happens to be Blank Canvas. However, the theme name is translated across locales, so if we specify it, it would only work for the English run.

Reply:
I see, the `selectDesign` should work with non-English strings, by the way. 
I'm guessing the design that's actually selected doesn't matter for the purpose of the test.